keyboards/model_f_labs/xwhatsit_util_comm.h

```diff
@@ -0,0 +1,64 @@
+/* Copyright 2020 Purdea Andrei
+ * Copyright 2021 Matthew J Wolf
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 2 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+#pragma once
+
+#ifndef RAW_EPSIZE
+#    define RAW_EPSIZE 32
+#endif
+
+#ifndef UTIL_COMM_H
+#    define UTIL_COMM_H
```

Comment:
`#pragma once` takes care of this.
```suggestion
```

Reply:
It's not being used in keyboards/model_f_labs/xwhatsit_util_comm.c. I removing it.